patches/rrweb@2.0.0-alpha.13.patch

```diff
@@ -528,30 +528,248 @@ index 22fee601e786c1d8dfb5c01d2e359c8bcbac7c42..20c3e14adfde860563e8dd902041bd14
+-            this.start = start;
+-            this.end = { line: lineno, column };
+-            this.source = options.source;
+-        }
+-    }
+-    Position.prototype.content = css;
+-    const errorsList = [];
+-    function error(msg) {
+-        const err = new Error(`${options.source || ''}:${lineno}:${column}: ${msg}`);
+-        err.reason = msg;
+-        err.filename = options.source;
+-        err.line = lineno;
+-        err.column = column;
+-        err.source = css;
+-        if (options.silent) {
+-            errorsList.push(err);
+-        }
+-        else {
+-            throw err;
+-        }
+-    }
+-    function stylesheet() {
+-        const rulesList = rules();
+-        return {
+-            type: 'stylesheet',
+-            stylesheet: {
+-                source: options.source,
+-                rules: rulesList,
+-                parsingErrors: errorsList,
+-            },
+-        };
+-    }
+-    function open() {
+-        return match(/^{\s*/);
+-    }
+-    function close() {
+-        return match(/^}/);
+-    }
+-    function rules() {
+-        let node;
+-        const rules = [];
+-        whitespace();
+-        comments(rules);
+-        while (css.length && css.charAt(0) !== '}' && (node = atrule() || rule())) {
+-            if (node) {
+-                rules.push(node);
+-                comments(rules);
+-            }
+-        }
+-        return rules;
+-    }
+-    function match(re) {
+-        const m = re.exec(css);
+-        if (!m) {
+-            return;
+-        }
+-        const str = m[0];
+-        updatePosition(str);
+-        css = css.slice(str.length);
+-        return m;
+-    }
+-    function whitespace() {
+-        match(/^\s*/);
+-    }
+-    function comments(rules = []) {
+-        let c;
+-        while ((c = comment())) {
+-            if (c) {
+-                rules.push(c);
+-            }
+-            c = comment();
+-        }
+-        return rules;
+-    }
+-    function comment() {
+-        const pos = position();
+-        if ('/' !== css.charAt(0) || '*' !== css.charAt(1)) {
+-            return;
+-        }
+-        let i = 2;
+-        while ('' !== css.charAt(i) &&
+-            ('*' !== css.charAt(i) || '/' !== css.charAt(i + 1))) {
+-            ++i;
+-        }
+-        i += 2;
+-        if ('' === css.charAt(i - 1)) {
+-            return error('End of comment missing');
+-        }
+-        const str = css.slice(2, i - 2);
+-        column += 2;
+-        updatePosition(str);
+-        css = css.slice(i);
+-        column += 2;
+-        return pos({
+-            type: 'comment',
+-            comment: str,
+-        });
+-    }
+-    function selector() {
 -        whitespace();
 -        while (css[0] == '}') {
 -            error('extra closing bracket');
 -            css = css.slice(1);
 -            whitespace();
 -        }
 -        const m = match(/^(("(?:\\"|[^"])*"|'(?:\\'|[^'])*'|[^{])+)/);
-+        const m = match(/^([^{]+)/);
```

Comment:
Diff gets a little funky here because we are removing a previously applied patch